### PR TITLE
feat(loading): Add request data for empty logic display and reuse

### PR DESCRIPTION
**Pull request by 1063428973**
#

https://user-images.githubusercontent.com/32738523/107175359-914d2700-6a07-11eb-9ce2-bc68512af3ef.mp4

### CODE REVIEW

File: packages/loading/src/createLoadingComponent.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { createVNode, h, reactive, ref, render, toRefs, Transition, VNode, vShow, withCtx, withDirectives } from 'vue'
+import { createVNode, h, reactive, ref, render, toRefs, Transition, VNode, vShow, withCtx, withDirectives, defineComponent } from 'vue'
 import { removeClass } from '@element-plus/utils/dom'
+import Empty from '../../empty/src/index.vue'
```
Comment (nurlansu) [inline packages/loading/src/createLoadingComponent.ts:3]: Can't we use global import here?

```suggestion
import Empty from '@element-plus/empty'
```

File: packages/loading/src/createLoadingComponent.ts [hunk 2]
```diff
@@ -87,31 +92,49 @@ export function createLoadingComponent({
 
       const spinnerText = h('p', { class: 'el-loading-text' }, [this.text])
 
+      const loadingContent = h('div', {
+        class: 'el-loading-spinner',
+      }, [
+        !this.spinner ? spinner : noSpinner,
```
Comment (nurlansu) [inline packages/loading/src/createLoadingComponent.ts:98]: Can you change the order for better reading?

```suggestion
        this.spinner ? noSpinner : spinner,
```

**Review by suyundukov — CHANGES_REQUESTED** [review]

**Comment by Ryan2128** [comment]
Sorry, I think this pr should not be merged, beacuse we should only provide atomic component.

Your requirements can be obtained by combining loading and emptying, instead of adding new functions to atomic components.

What's your advice? @JeremyWuuuuu

**Comment by kooriookami** [comment]
Thanks for your PR, but this might not a good change. So I close it.

**Pull request closed without merging**